test_pgeocode.py

```diff
@@ -1,182 +0,0 @@
-# -*- coding: utf8 -*-
```

Comment:
This file should not be removed.

Reply:
Yes, I got confused. Thanks for responding, I'm happy to help. I will make the necessary adjustments.